Sources/scipio/CreateCommands.swift

```diff
@@ -1,11 +1,11 @@
 import Foundation
 import ScipioKit
-import ArgumentParser
+@preconcurrency import ArgumentParser
```

Comment:
SwiftPM still depends on an old swift-argument-parser. It doesn't support concurrency, so we have to annotate `preconcurrency`

Reply:
Question: What happens if there is no annotate `preconcurrency`?

Reply:
`CommandConfiguration` should be `Sendable`, but this version doesn't support it yet.